### PR TITLE
fix(emoji): Use colorful version of warning emoji

### PR DESCRIPTION
**Pull request by zharinov**
## Changes

- patch emojibase data for GitHub shortcodes: use colorful `⚠️` instead of `⚠` (which looks cheap)
- adjust the tranform logic accordingly
- write tests to make sure it's safe

**Warning: this PR could cause lots of updates for existing PRs**

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: lib/modules/platform/github/index.ts [hunk 1]
```diff
@@ -1950,6 +1950,7 @@ export function massageMarkdown(input: string): string {
     .replace(regEx(/]: https:\/\/github\.com\//g), ']: https://togithub.com/')
     .replace('> ℹ **Note**\n> \n', '> [!NOTE]\n')
```
Comment (viceice) [inline lib/modules/platform/github/index.ts:1951]: is there also a colorful info emoji?

File: lib/util/emoji.ts [hunk 2]
```diff
@@ -37,6 +43,7 @@ function lazyInitMappings(): void {
     for (const [hex, val] of Object.entries(result.data)) {
       const shortCodes = is.array(val) ? val : [val];
       shortCodesByHex.set(hex, `:${shortCodes[0]}:`);
+      shortCodesByHex.set(stripHexCode(hex), `:${shortCodes[0]}:`);
```
Comment (viceice) [inline lib/util/emoji.ts:46]: what is this doing? 🤔

Comment (zharinov) [reply lib/util/emoji.ts:46]: It allows to convert both `26A0-FE0F` (⚠️) and `26A0` (⚠) back to the `:warning:`

File: lib/modules/platform/github/index.ts (hunk 1 above)
Comment (zharinov) [reply lib/modules/platform/github/index.ts:1951]: Yes, but I guess we don't care about other. I see only warning one behaving weird.

**Review by viceice — DISMISSED** [review]

**Review by viceice — APPROVED** [review]

File: lib/util/emoji.spec.ts [hunk 3]
```diff
@@ -21,6 +21,11 @@ describe('util/emoji', () => {
       expect(emojify(':foo: :bar: :bee:')).toBe(':foo: :bar: 🐝');
     });
 
+    it('convert warning shortcode to emoji', () => {
```
Comment (HonkingGoose) [inline lib/util/emoji.spec.ts:24]: Silly question incoming! 🫠 

Can we use the _good colorful emojis_ in the source files _directly_? So we can drop the _convert bad emoji to good emoji_ code? It feels a bit weird to have all this code dealing with emojis.

Comment (zharinov) [reply lib/util/emoji.spec.ts:24]: No, because source file is located in the `node_modules`

**Pull request merged**